.github/workflows/pipeline.yml

```diff
@@ -149,6 +149,13 @@ jobs:
                   node-version: 18.x
             - name: Install pnpm modules
               run: pnpm install
+            - name: Replace placeholder with instrumentation key
```

Comment:
There are major changes required to the whole concept used here.

Reply:
1. it is embedding the internal SAP tools-suite dev telemetry key into the generated artifact - if a real telemetry key is required a new separate key/azure project is required
2. some separation of dev/test keys versus production keys are required - real consumer such as tools-suite will need to add its dev or production keys
3. the replacement cannot be based on key value - but has to ignore the value
4. If this is for test only then the test such just read the secret as an ENV var - the code to read and inject an env var is straight forward
5. the tools-suite patch for developers could be used but requires a new azure key

Reply:
As we aligned in Architecture round, telemetry will be initialized from toolsuite, open ux changes related to telemetry initialization will be removed